_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_vaapi.h

```diff
@@ -297,6 +297,8 @@ mfxStatus SetSkipFrame(
 
         UMC::Mutex m_guard;
         HeaderPacker m_headerPacker;
+
+        mfxU32 m_mbbrc;
```

Comment:
I see zero reasons to make it class-member. In both cases when it is used (CreateAccelerationService and Reset), its' value constructed/assigned right before usage and the stored one is ignored.